### ISSUE TITLE
Powered Draw Bridge oddity?

### ISSUE THREAD

**Issue by SonicX8000** [MC-U-Team/Draw-Bridge#4]
I was going to suggest a linked draw bridge kind of thing where if you have draw bridge blocks connecting to one another... they would all power on from one redstone source. That's kind there... here's an image with 3 drawbridge blocks with 3 iron bars in each of them.

![](https://cdn.discordapp.com/attachments/377171551155650560/533253233443864587/drawbridgeredstone.PNG)

"Left"
The draw bridge block in the middle is powered by the actual redstone and apparently it powers up the blocks on the sides allowing a 3 block door, if you wish.

"Right"
The draw bridge block in the middle is powered by the block that the redstone is placed on... and only one places the blocks that are stored while the other two are not powered.

Unsure if it's a quirk with the redstone or maybe I'm missing something.  There's no hidden redstone in the works in the picture.

If this is just vanilla minecraft doing it's think... don't suppose linked drawbridges could become a thing? Could be neat to have say a hanger doorway of some sorts or a 3x3 door made of glass or something.

**Comment by MrTroble** [comment]
This seams to be a good idea maybe we will implement that.
I will look at this asap and discuss it with the others!

Thank you for your detailed feedback.
Cheers
~MrTroble

**Comment by MrTroble** [comment]
I have added it to the repo for test.

I will see how it turns out it currently has a range of to blocks and is disabled by the "Need Redstone" switch which is not the way to go. I will probably add an extra switch for that.

Cheers
~MrTroble

**Comment by MrTroble** [comment]
c5f649e44c5804090a0165f095556a5a2849ef75

This has been added with a toggle! 
The performance has been taken care of!

Thanks again.

Cheers again
~MrTroble

**Comment by SonicX8000** [comment]
Nice, so if one draw bridge is on the neighboring drawbridges will turn on as well with that toggle?

**Comment by HyCraftHD** [comment]
Currently yes. We have encountered some performance issues so this build is still a beta.
We are looking more into this feature soon.

**Comment by SonicX8000** [comment]
Not trying to rush but just wondering on how this is going. One more suggestion if I may... could there be a sound effect when the drawbridge activates/deactivates? Like maybe how the vanilla minecraft pistons sound occur when it powers on/off.

If it's too much of an hassle with the performance issues, I got the drawbridge working how I wanted with the use of Redstone Paste from the "Redstone Paste" mod so I was able to do some custom doors with the way the draw bridge places the blocks. It could still be an option but that's if it doesn't cause any drawbacks.
